README.md

```diff
@@ -36,7 +36,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Is this equivalent to [this from the demo repo](https://github.com/helm/charts-repo-actions-demo/blob/60436cba8c31fc0d8d22953c67cf1ceae5585323/.github/workflows/release.yaml#L15-L16)?

```yaml
      - name: Fetch history
        run: git fetch --prune --unshallow
```

If so, it seems a bit nicer. Would you be into making a PR over there as well?

Reply:
Sure